content/docs/dvclive/live/index.md

```diff
@@ -63,6 +64,9 @@ You can use `Live()` as a context manager. When exiting the context manager,
   (`.html`) or Markdown (`.md`) depending on the value of the `report`
   parameter.
 
+- `system_monitor` - a [`SystemMonitor`](doc/dvclive/monitor_system) instance if
+  the `monitor_system` parameter is `True`.
```

Comment:
A couple questions on this:
1. Is there a reason you use the kwarg `monitor_system` but the property is `live.system_monitor` and the class is `SystemMonitor`? I keep getting confused.
2. WDYT about making `live.system_monitor` a method instead of a property so that it can fit with the other methods instead of adding another section? The section calls attention to this new feature, but I slightly worry it will feel out of place once it's not so new and in a separate place from the rest of the API.

Reply:
> Is there a reason you use the kwarg monitor_system but the property is live.system_monitor and the class is SystemMonitor? I keep getting confused.

I came from @shcheklein comment on how `monitor_system` is a `bool` and how `monitor_...` looks like `bool` after that. 

> WDYT about making live.system_monitor a method instead of a property so that it can fit with the other methods instead of adding another section? The section calls attention to this new feature, but I slightly worry it will feel out of place once it's not so new and in a separate place from the rest of the API.

I think the ideal behavior would be: 
```py
live = Live(monitor_system=False)
live.monitor_system(interval=0.1)
```

It fixed both of your comments. I was a bit blinded by my callback stuff. But it seems easier this way, you're right. 
The only thing I'm not sure about is the naming of the method. Should we go for `monitor_system` or `system_monitor`. I feel like `monitor_system` is clearer; it starts with a verb, and you can associate it with the `Live` argument readily. But as Ivan said, both a `bool` and a method would share a name but have different types, and it can be confusing.

Reply:
I made changes to have a method, it is indeed clearer in the code and the doc. I took the decision to go for `monitor_system` but we can agree on another name if that is weird for you.